network/endpoint_windows.go

```diff
@@ -281,14 +282,28 @@ func (nw *network) configureHcnEndpoint(epInfo *EndpointInfo) (*hcn.HostComputeE
 }
 
 func (nw *network) deleteHostNCApipaEndpoint(cli apipaClient, networkContainerID string) error {
```

Comment:
should be endpoint function

Reply:
the create and delete HostNCApipaEndpoint functions were already as `(nw *network)` so I don't want to refactor too much in this PR, I would prefer to leave this change for later

Reply:
hmm right..should ve been endpoint function..that' fine..let it be